### PR TITLE
doc/rbd: refine "Create a Block Device User"

### PR DESCRIPTION
**Pull request by zdover23**
Refine "Create a Block Device User" in doc/rbd/rados-rbd-cmds.rst.

https://tracker.ceph.com/issues/57001

Signed-off-by: Zac Dover <zac.dover@gmail.com>

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: doc/rbd/rados-rbd-cmds.rst [hunk 1]
```diff
@@ -28,26 +28,29 @@ Create a Block Device Pool
 Create a Block Device User
 ==========================
 
-Unless specified, the ``rbd`` command will access the Ceph cluster using the ID
-``admin``. This ID allows full administrative access to the cluster. It is
-recommended that you utilize a more restricted user wherever possible.
+Unless otherwise specified, the ``rbd`` command uses the ID ``admin`` to access the Ceph cluster. The ``admin`` ID allows full administrative access to the cluster. We recommend that you acess the Ceph cluster with an ID that has fewer permissions than the ``admin`` ID does. We call this non-``admin`` ID a "block device user" or "Ceph user".
```
Comment (idryomov) [inline doc/rbd/rados-rbd-cmds.rst:31]: Instead of bare ID, I'd use "user ID" everywhere.

Comment (anthonyeleven) [reply doc/rbd/rados-rbd-cmds.rst:31]: I might suggest "Ceph user" instead, I think "user ID" is even more likely to be confused with Linux /etc/passwd-type users.

File: doc/rbd/rados-rbd-cmds.rst [hunk 2]
```diff
@@ -28,26 +28,29 @@ Create a Block Device Pool
...
 
-To `create a Ceph user`_, with ``ceph`` specify the ``auth get-or-create``
-command, user name, monitor caps, and OSD caps::
+To `create a Ceph user`_, use the ``ceph`` command with the ``auth
```
[the remark this answers is not in the document]
Comment (anthonyeleven) [reply doc/rbd/rados-rbd-cmds.rst:33]: agree

File: doc/rbd/rados-rbd-cmds.rst [hunk 3]
```diff
@@ -28,26 +28,29 @@ Create a Block Device Pool
...
+get-or-create`` command to specify user name, monitor caps (capabilities), and
+OSD caps (capabilities):
 
-        ceph auth get-or-create client.{ID} mon 'profile rbd' osd 'profile {profile name} [pool={pool-name}][, profile ...]' mgr 'profile rbd [pool={pool-name}]'
+.. prompt:: bash #
```
[the remark this answers is not in the document]
Comment (anthonyeleven) [reply doc/rbd/rados-rbd-cmds.rst:37]: Do we think most people won't be running as root?

**Review by anthonyeleven — APPROVED** [review]
non-blocking suggestions

File: doc/rbd/rados-rbd-cmds.rst (hunk 3 above)
[the remark this answers is not in the document]
Comment (zdover23) [reply doc/rbd/rados-rbd-cmds.rst]: %s/bash #/bash $/g - this change is in now.

File: doc/rbd/rados-rbd-cmds.rst (hunk 1 above)
Comment (zdover23) [reply doc/rbd/rados-rbd-cmds.rst:31]: I'm going to go with "Ceph user ID" for now, for minimum ambiguity.

**Review by anthonyeleven — APPROVED** [review]

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/49300 - Quincy backport
https://github.com/ceph/ceph/pull/49301 - Pacific backport

**Pull request merged**